p2p/pex_reactor.go

```diff
@@ -293,6 +350,18 @@ func (r *PEXReactor) ensurePeers() {
 	}
 }
 
+// addrToSrcPeer return the Peer that provided the address in the param
+func (r *PEXReactor) addrToSrcPeer(addr *NetAddress) Peer {
+	peers := r.Switch.Peers().List()
```

Comment:
so this will only return active peers.

should we consider marking good for currently inactive peers if they were the source of a peer we just dialed?

Reply:
The trust metric implementation does not continue tracking peers that are no longer active